client-apis/decrypt.md

```diff
@@ -162,6 +164,8 @@ that implementation SHOULD NOT provide an API that allows the caller to stream t
 
 The [encryption context](../framework/structures.md#encryption-context) that is used as
 additional authenticated data during the decryption of the input [encrypted message](#encrypted-message).
+Specifically, it MUST be the union of the encryption context serialized into the message header and
+the encryption context for authentication only, if available.
```

Comment:
`encryption context for authentication only` Can we point to the definition of this somewhere?